project.clj

```diff
@@ -36,7 +38,7 @@
                  [day8.re-frame/forward-events-fx "0.0.6"]
 
                  ;; District General Libraries
-                 [district0x/async-helpers "0.1.3"]
+                 [district0x/async-helpers "0.1.3-SNAPSHOT"]
```

Comment:
can we avoid SNAPSHOT versions in production releases? They can lead to hard-to-find bugs and should be reserved for local development. Most of our repositories are configured to release non-snapshots.

Reply:
done